dashboard/test/ui/runner.rb

```diff
@@ -677,6 +677,12 @@ def run_feature(browser, feature, options)
   # BrowserStack was reporting Windows 6.0 and 6.1, causing different baselines
   run_environment['APPLITOOLS_HOST_OS'] = 'Windows 6x' unless browser['mobile']
 
+  # The "view more" links in the public project gallery for App Lab and Game Lab
+  # are controlled by DCDO so we can quickly hide them if there are
+  # inappropriate projects. This enviroment lets us test "view more" link
+  # visibility without updating the tests every time we toggle the flag.
+  run_environment['HIDE_VIEW_MORE_LINKS'] = DCDO.get('image_moderation', {})['limited_project_gallery'].to_s
```

Comment:
Given that the environment entries are global to all tests, it might be good to name it more explicitly than `HIDE_VIEW_MORE_LINKS`.  How about `IMAGE_MODERATION_PUBLIC_GALLERY_LINK_VISIBILITY`?  Or how about recording the entire contents of the `image_moderation` hash and then examining the `limited_project_gallery` value in the test step?

Reply:
It'd be nice not to need this at all.  I assume you've already explored this, but what prevents us from referring to the DCDO config directly in the steps file?